test/reporters/tests/reporters.spec.ts

```diff
@@ -83,6 +103,34 @@ test('JUnit reporter with outputFile', async() => {
   rmSync(outputFile)
 })
 
+test('JUnit reporter with outputFile object', async() => {
+  // Arrange
+  const reporter = new JUnitReporter()
+  const outputFile = resolve('report.xml')
```

Comment:
Each of the `outputFile`'s should have unique names in tests. Also the `rootDirectory`'s should be unique. 
Earlier I saw that when tests were run on CI, some tests were locking the files when removing them, while other tests were trying to create them. Easy solution was to use unique names in each test so that they don't conflict.

Reply:
makes sense tbh, gonna change that

Reply:
fixed it, thx for the hint :D